pkg/tests/dbaas_operator_tests.go

```diff
@@ -3,8 +3,13 @@ package tests
 import (
 	"context"
 	"errors"
-	"flag"
 	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
```

Comment:
Curious why these are getting rearranged

Reply:
This was done by IDE (Visual Studio Code) using gofmt automatically so that the entries are organized in a standard way (core modules, 3rd party modules, and then modules of the current project).

Reply:
OK. I updated the Makefile so that the build will check the format first. It uses goimports that is commonly used in the community.

Reply:
This does not format the code, but instead fails the build if not formatted according to what it expects, and without providing a good / clear message. That's an issue for IntelliJ users among us!